content/docs/studio/user-guide/model-registry/add-a-model.md

```diff
@@ -57,4 +57,5 @@ https://www.youtube.com/watch?v=szzv4ZXmYAs
    base branch.
 
 [gto]: https://github.com/iterative/gto
+[from the cli]: https://github.com/iterative/gto#versioning-an-artifact
```

Comment:
Is that the right link?

Reply:
No. it's https://github.com/iterative/gto#annotating